bitnami/minio/templates/provisioning-job.yaml

```diff
@@ -273,6 +273,12 @@ spec:
           resources: {{- include "common.resources.preset" (dict "type" .Values.provisioning.resourcesPreset) | nindent 12 }}
           {{- end }}
           volumeMounts:
+            - name: empty-dir
```

Comment:
I see some `mc admin service restart` in the provisioning. Also the command is customizable. What if we also  mount `/opt/bitnami/minio/tmp` just in case?